src/components/draw/DrawingToolBar.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import eraser from "../../assets/drawingTools/eraser.png";
+import redPencil from "../../assets/drawingTools/colorPencil_red.png";
+import orangePencil from "../../assets/drawingTools/colorPencil_orange.png";
+import yellowPencil from "../../assets/drawingTools/colorPencil_yellow.png";
+import greenPencil from "../../assets/drawingTools/colorPencil_green.png";
+import bluePencil from "../../assets/drawingTools/colorPencil_blue.png";
+import navyPencil from "../../assets/drawingTools/colorPencil_navy.png";
+import purplePencil from "../../assets/drawingTools/colorPencil_purple.png";
+import pinkPencil from "../../assets/drawingTools/colorPencil_pink.png";
+import rainbowPencil from "../../assets/drawingTools/colorPencil_rainbow.png";
+
+const DrawingToolBar = ({ setSelectedColor, setColorPickerOpen }) => {
+  const [selectedTool, setSelectedTool] = useState(1);
+
+  const toolDatas = [
+    { tool: eraser, color: "transparent" },
+    { tool: redPencil, color: "#FF0900" },
+    { tool: orangePencil, color: "#FF7700" },
+    { tool: yellowPencil, color: "#FFF200" },
+    { tool: greenPencil, color: "#88FF00" },
+    { tool: bluePencil, color: "#0051FF" },
+    { tool: navyPencil, color: "#0C00AD" },
+    { tool: purplePencil, color: "#9000FF" },
+    { tool: pinkPencil, color: "#FF00EA" },
+    { tool: redPencil, color: "#FF0900" },
+    { tool: orangePencil, color: "#FF7700" },
+    { tool: yellowPencil, color: "#FFF200" },
+    { tool: greenPencil, color: "#88FF00" },
+    { tool: bluePencil, color: "#0051FF" },
+    { tool: navyPencil, color: "#0C00AD" },
+    { tool: purplePencil, color: "#9000FF" },
+    { tool: pinkPencil, color: "#FF00EA" },
+    {
+      tool: rainbowPencil,
+    },
+  ];
+
+  // rainbowPencil 클릭 시 컬러피커 표시
+  const handleToolSelect = (index) => {
+    setSelectedTool(index);
+    if (index === toolDatas.length - 1) {
+      setColorPickerOpen(true);
+    } else {
+      setSelectedColor(toolDatas[index].color);
+      setColorPickerOpen(false);
+    }
+  };
```

Comment:
이 함수를 누가 읽었을때 마지막 인데스에 무엇이 들었는지 알수 있게 해주면 좋을것 같아요
```suggestion
  const RAINBOW_PENCIL_ID = "rainbow";

  const handleToolSelect = (clickedPencil) => {
    setSelectedTool(clickedPencil);
    if (clickedPencil.id === RAINBOW_PENCIL_ID) {
      setColorPickerOpen(true);
      return;
    }
    setSelectedColor(clickedPencil.color);
  };
```

무조건 이렇게 하라는건 아니고 미팅에서 따로 볼게요

Reply:
이렇게 하는게 훨씬 이해하기 좋네요. 수정했습니당